README.md

```diff
@@ -1,163 +1,257 @@
 
 ```
-source <(tw generate-completion)
+$ tw credentials create aws -n aws -a <aws access key> -s <aws secret key>
 ```
 
-Check available create credentials commands
+### 6. List Credentials
+
+List the credentials available in the workspace.
 
 ```
-tw credentials create -h
+$ tw credentials list
 ```
 
-Check help and create AWS credentials
+### 7. Provision Compute Environment
+
+Create a Compute Environment for AWS Batch with automatic provisioning of cloud compute resources. 
 
 ```
-tw credentials create aws -h
-tw credentials create aws -n aws -a <your aws access key> -s <your aws secret key> 
-tw credentials list
+$ tw compute-envs create aws -n aws-ce -r eu-west-1 --max-cpus=256 -w s3://<bucket-name> 
 ```
 
-Create an AWS compute environment with automatic provisioning of compute resources
+* This create all the required AWS Batch resources in the AWS Ireland region (eu-west-1) with a maximum total of 256 CPUs. 
+
+* An existing S3 bucket will be used as a work directory. 
+
+* See the [user documentation](https://help.tower.nf/compute-envs/aws-batch/#forge) for complete details of Tower Forge.
+
+
+### 8. Create a Pipeline
+
+Add a pre-configured workflow that can be reused. 
 
 ```
-tw compute-envs create aws -n demo -r eu-west-1 -w s3://nextflow-ci/jordeu --max-cpus=123 --fusion
+$ tw pipelines create -n sleepy-flow --params=<(echo 'timeout: 60') https://github.com/pditommaso/nf-sleep
 ```
 
-Create a new pipeline at launchpad
-_NOTE: the params option has to be a file. Here we are using Bash pipes to convert a command output into a file automatically._
+*The `params` option should be a YAML or JSON file. Here we use a Bash pipe to convert a command into a YAML file automatically.*
+
+Pipelines consists of a workflow repository, launch parameters, and a Compute Environment. When no Compute Environment is specified, the primary CE is used.
+
+
+### 9. Launch it! :rocket:
+
+Run the workflow.
 
 ```
-tw pipelines create -n sleep_one_minute --params=<(echo 'timeout: 60') https://github.com/pditommaso/nf-sleep
+$ tw launch sleepy-flow
 ```
 
-Run it!
+### 10. Change launch parameters
+
+Launch the workflow with different parameters.
 
 ```
-tw launch sleep_one_minute
+$ tw launch sleepy-flow --params=<(echo 'timeout: 30')
 ```
 
-Run it with different parameters
+### 11. Update a Pipeline
+
+Pipelines can be modified with new launch parameters using the `update` command. 
 
 ```
-tw launch sleep_one_minute --params=<(echo 'timeout: 30')
+$ tw pipelines update -n sleepy-flow --params=<(echo 'timeout: 30')
 ```
 
-Directly launch a Github pipeline not defined at launchpad
+### 12. Launch a workflow directly
+
+It possible to directly launch Git workflows that have not been added to a workspace as Pipelines.
 
 ```
-tw launch nextflow-io/hello
+$ tw launch nextflow-io/hello
 ```
 
-Update a launchpad pipeline
+### :gift: Bonus: Activate autocomplete 
+
+Install autocomplete in your current session, type `tw pi` then press tab twice.
```

Comment:
Here I understand that I've to:

1. type `tw pi`
2. press tab twice
3. Execute `source <(tw generate-completion)`

Maybe it's better to first show the command to run and then explain how to use autocomplete.